studybuddies/src/app/services/httperror-interceptor.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { HTTPErrorInterceptorService } from './httperror-interceptor.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202914)
```suggestion
import { HTTPErrorInterceptorService } from "./httperror-interceptor.service";
```